images/ansible/roles/kubernetes/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- import_tasks: debian.yml
+  when: ansible_os_family == "Debian"
+
+- import_tasks: redhat.yml
+  when: ansible_os_family == "RedHat"
+
+- name: ensure that the kubelet is running
+  service:
+    name: kubelet
+    state: started
+    enabled: true
+
+- name: cache docker images for kubeadm
+  docker_image:
+    name: "{{ item }}"
+  with_items: kubernetes_cached_images
+  when: kubernetes_enable_cached_images | bool
```

Comment:
Lets kill the cached image handling for now, especially since it is docker specific.